photon-server/src/main/resources/models/labels-robots.txt

```diff
@@ -0,0 +1,2 @@
+note
+robot
```

Comment:
This is wrong for the v5 model, which is still the default right?

Reply:
Yeah it should be reading from labels.txt and yolov5 by default, I should change the labels to have a _v5 and _v8 suffix...